proto/prometheus/v1/types.proto

```diff
@@ -0,0 +1,190 @@
+// Copyright 2017 Prometheus Team
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// NOTICE: THIS FILE HAS BEEN MODIFIED FROM THE ORIGINAL
```

Comment:
Could you provide some quick context of "how so?" maybe a link to the original you copied

Reply:
Added.